Flabbergasted-Flamingos/Bangazon-admin-app
# added training list view and training_add form view

# Description
adds list and add functionality for the trainings resource

## Related Ticket(s)
closes #9 

## Expected Behavior
When you navigate to trainings/ you should see a list of trainings in the database. 
Add training button on the bottom should redirect to the form to add a training, that submittal redirects back to the list, new training should appear at the bottom. 

## Steps to Test Solution

```
git fetch --all 
git checkout trainings_detail_view
```
remigrate, run seeder. 

```
python manage.py runserver
```

navigate to trainings/, look for current trainings. 
click "add training", fill out form, submit
look at list of trainings to ensure that the new training is in the database.
